types/module/simulation/simulation.go

```diff
@@ -0,0 +1,67 @@
+package simulation
+
+import (
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/types/simulation"
+)
+
+// AppSimulationManager defines a simulation manager that provides the high level utility
+// for managing and executing simulation functionalities for a group of modules
+type AppSimulationManager struct {
```

Comment:
why do we need a different simulation manager than the one already defined in cosmos-sdk?